ecl2df/compdat.py

```diff
@@ -978,7 +978,11 @@ def compdat_main(args):
     write_dframe_stdout_file(compdat_df, args.output, index=False, caller_logger=logger)
 
 
-def df(eclfiles: EclFiles, initvectors: Optional[List[str]] = None) -> pd.DataFrame:
+def df(
+    eclfiles: EclFiles,
+    initvectors: Optional[List[str]] = None,
+    zonemap: Optional[Dict[int, str]] = None,
```

Comment:
Can you add a description of the arguments in the docstring (stating that this option for `zonemap` is for overriding the zonemap it otherwise will extract from `eclfiles`)